easybuild/easyblocks/l/lammps.py

```diff
@@ -121,6 +121,9 @@ def prepare_step(self, *args, **kwargs):
 
     def configure_step(self, **kwargs):
         """Custom configuration procedure for LAMMPS."""
+        vtk = get_software_root('VTK')
+        if not vtk:
```

Comment:
If you don't use "vtk" any other place there is no reason to create a variable for the result just do
if not get_software_root('VTK')

Reply:
yup